src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/transformation/TransformationClientTest.java

```diff
@@ -90,9 +91,9 @@ public void should_throw_invalid_data_exception_for_unprocessable_entity_respons
                 .willReturn(aResponse().withBody(errorResponse().toString()).withStatus(UNPROCESSABLE_ENTITY.value())));
 
         // when
-        InvalidCaseDataException exception = catchThrowableOfType(
+        UnprocessableEntityException exception = catchThrowableOfType(
             () -> client.transformExceptionRecord(url(), exceptionRecordRequestData(), s2sToken),
-            InvalidCaseDataException.class
```

Comment:
Having introduced `UnprocessableEntityException`, we should rename this one to `BadRequestException`, so that  the would be no doubt about its purpose.

Reply:
Never mind